package.json

```diff
@@ -9,13 +9,12 @@
     "test-packages/*"
   ],
   "scripts": {
-    "test": "mocha test-packages/*-test.js"
+    "test": "jest --runInBand"
```

Comment:
Do we need in bad due to mutation of the local repo contents?

Reply:
yep, and to prevent concurrent access of the git cleanup commands. there are probably ways around that, but for now this should be sufficient.